docs/src/bootstrap.md

```diff
@@ -25,49 +25,45 @@ For example, a simple linear mixed-effects model for the `Dyestuff` data in the
 package for [`R`](https://www.r-project.org) is fit by
 
 ```@example Main
-using DataFrames, Gadfly, MixedModels, Random
+using DataFrames
+using DataFramesMeta  # dplyr-like operations
+using Gadfly          # plotting package
+using MixedModels
+using Random
 ```
 
 ```@example Main
 dyestuff = MixedModels.dataset(:dyestuff)
 m1 = fit(MixedModel, @formula(yield ~ 1 + (1 | batch)), dyestuff)
 ```
 
-To bootstrap the model parameters, first initialize a random number generator
+To bootstrap the model parameters, first initialize a random number generator then create a bootstrap sample
 
 ```@example Main
 const rng = MersenneTwister(1234321);
+samp = parametricbootstrap(rng, 10_000, m1, use_threads=true);
+df = DataFrame(samp.allpars);
+first(df, 10)
 ```
 
-then create a bootstrap sample
+Especially for those with a background in [`R`](https://www.R-project.org/) or [`pandas`](https://pandas.pydata.org),
+the simplest way of accessing the parameter estimates in the parametric bootstrap object is to create a `DataFrame` from the `allpars` property as shown above.
 
+The [`DataFramesMeta`](https://github.com/JuliaData/DataFramesMeta.jl) package provides macros for extracting rows or columns of a dataframe.
+A density plot of the estimates of `σ`, the residual standard deviation, can be created as
 ```@example Main
-samp = parametricbootstrap(rng, 10_000, m1);
-propertynames(samp)
-```
-
-As shown above, the sample has several named properties, which allow for convenient extraction of information.  For example, a density plot of the estimates of `σ`, the residual standard deviation, can be created as
-```@example Main
-plot(x=samp.σ, Geom.density, Guide.xlabel("Parametric bootstrap estimates of σ"))
+σres = @where(df, :type .== "σ", :group .== "residual").value
+plot(x = σres, Geom.density, Guide.xlabel("Parametric bootstrap estimates of σ"))
 ```
 For the estimates of the intercept parameter, the `getproperty` extractor must be used
```

Comment:
There is no explicit call to `getproperty` here, which might be confusing to Julia neophytes (which the rest of the style seems to assume). I don't have a better suggestion at the moment, but this is a spot where we might be able to add more polish in the future.